extensions/ext/GLSL_EXT_integer_dot_product.txt

```diff
@@ -0,0 +1,288 @@
+Name
+
+    EXT_integer_dot_product
+
+Name Strings
+
+    GL_EXT_integer_dot_product
+
+Contact
+
+    Jeff Bolz, NVIDIA (jbolz 'at' nvidia.com)
+
+Contributors
+
+    Jeff Bolz, NVIDIA (jbolz 'at' nvidia.com)
+    Kevin Petit, Arm Ltd. (kevin.petit 'at' arm.com)
+    Tobias Hector, AMD
+    Graeme Leese, Broadcom Corporation
+    Georg Lehmann
+
+Status
+
+    Complete.
+
+Version
+
+    Last Modified:      October 3, 2022
+    Revision:           1
+
+Dependencies
+
+    This extension can be applied to OpenGL GLSL versions 4.50
+    (#version 450) and higher.
+
+    This extension can be applied to OpenGL ES ESSL versions 3.20
```

Comment:
Is there any reason why this requires ES3.2? Could it not be applied to version 300?

Reply:
I can't think of one, likely due to copy/paste. I'll change it to say 3.00.